src/middleware/devtools.ts

```diff
@@ -1,4 +1,5 @@
 import type {} from '@redux-devtools/extension'
+import type { ActionCreatorObject } from '@redux-devtools/utils';
```

Comment:
Doesn't this require all devtools users to install the utils lib? I'd like to make it opt-in. Could we use `import()` syntax to avoid it and if it's not installed it's just `any`?

Reply:
Please make sure that users can use `devtools` without installing `@redux-devtools/utils`, and typescript code compiles.

Reply:
As mentioned in the previous comment, as the type definition or the import never get to dist files code compiles. At least in the tests I have run. Only way I see this not compiling would be with a locally linked version of the repo and updating the package.json to import from the .ts file instead of the dist files.

So far, all works as expected.